diff --git a/vm_basic_types.h b/vm_basic_types.h
index 793ed5f..a59d3d6 100644
--- a/vm_basic_types.h
+++ b/vm_basic_types.h
@@ -770,12 +770,10 @@ typedef void * UserVA;
 #   endif
 
 #   if !defined(FROBOS) && !defined(_SSIZE_T) && !defined(ssize_t) && \
-       !defined(__ssize_t_defined) && !defined(_SSIZE_T_DECLARED) && \
-       !defined(_SSIZE_T_)
+       !defined(__ssize_t_defined) && !defined(_SSIZE_T_DECLARED)
 #      define _SSIZE_T
 #      define __ssize_t_defined
 #      define _SSIZE_T_DECLARED
-#      define _SSIZE_T_
 #      ifdef VM_I386
 #         ifdef VM_X86_64
              typedef int64 ssize_t;
